packages/manager/src/features/Profile/SSHKeys/CreateSSHKeyDrawer.tsx

```diff
@@ -45,6 +48,17 @@ const CreateSSHKeyDrawer = ({ open, onClose }: Props) => {
 
   const generalError = hasErrorFor('none');
 
+  const SSHTextAreaHelperText = () => (
+    <Typography component="span">
+      <Link to="https://www.linode.com/docs/guides/use-public-key-authentication-with-ssh/">
+        Learn about
+      </Link>{' '}
+      uploading an SSH key or generating a new key pair. Note that the public
+      key begins with <Code>ssh-rsa</Code> and ends with{' '}
+      <Code>your_username@hostname</Code>
```

Comment:
```suggestion
      <Code>your_username@hostname</Code>.
```

Can we add a period at the end of this sentence?

Reply:
committed your change!